.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.3.3
```

Comment:
@yshmarov  are you OK with introducing `asdf` [tools version](https://asdf-vm.com/) control?

Reply:
pls don't commit this. keep your dev environment config local

Reply:
removing now.